docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,72 @@
+[[proc_updating-booster-deployment-configuration]]
```

Comment:
The heading should be `[id='updating-the-deployment-configuration-of-a-booster_{context}']` (Let's use the `_{context}` suffix everywhere since it's likely going to be mandated by the guidelines.)